readme.md

```diff
@@ -53,8 +62,9 @@ console.log(results);
 ```
 
 Your Answer:
-```text
-
+```js
+var results = AuthorModel.find({name: "Bob"});
+console.log(name)
```

Comment:
The reason this wouldn't work is `.find` is asynchronous. So `console.log` would happen before the `.find` is done.